### PR TITLE
Fix `create-astro` on Windows

### PR DESCRIPTION
**Pull request by natemoo-re**
## Changes

- Skips Houston animations on Windows, which seem to be breaking the prompt
- Linked to #5227 but I'm not confident it fixes _all_ Windows issues

## Testing

Tested manually

## Docs

N/A, bug fix only

### CODE REVIEW

**Comment by natemoo-re** [comment]
!preview fixwin

**Comment by natemoo-re** [comment]
Confirmed via Windows users on Discord that this fixed it!

File: packages/create-astro/src/index.ts [hunk 1]
```diff
@@ -30,6 +31,8 @@ import { TEMPLATES } from './templates.js';
 // fixes the issue so that create-astro now works on all npm version.
 const cleanArgv = process.argv.filter((arg) => arg !== '--');
 const args = yargs(cleanArgv, { boolean: ['fancy'] });
+// Always skip Houston on Windows (for now)
+if (platform() === 'win32') args.skipHouston = true;
```
Comment (bholmesdev) [inline packages/create-astro/src/index.ts:35]: _sad houston beep boops_

**Review by bholmesdev — APPROVED** [review]
I'm sure windows peeps will appreciate this a hous-ton

**Pull request merged**